misc/deployments/staging.gno.land/gnoland.entrypoint.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env sh
+
+set -ex
+
+MONIKER=${MONIKER:-"gnode"}
+P2P_LADDR=${P2P_LADDR:-"tcp://0.0.0.0:26656"}
+RPC_LADDR=${RPC_LADDR:-"tcp://0.0.0.0:26657"}
+
+CHAIN_ID=${CHAIN_ID:-"dev"}
```

Comment:
Can we have this as `staging`?

Reply:
I think you set it in the compose, but we should update this default value as well